javascript/ql/src/semmle/javascript/Classes.qll

```diff
@@ -216,6 +216,13 @@ class ClassDefinition extends @classdefinition, ClassOrInterface, AST::ValueNode
     )
   }
 
+  /**
+   * Gets a direct super class, if it can be determined.
+   */
+  ClassDefinition getADirectSuperClass() {
```

Comment:
Not a fan of the name. Could we work "definition" into the name to disambiguate it with the existing (and badly named) `getSuperClass()`? Also, while it's technically true that there could be more than one direct super class, this is one of those cases where allowing for that possibility in the name and description causes more confusion than it's worth.